snap/snapcraft.yaml

```diff
@@ -0,0 +1,46 @@
+name: suspended-sentence
+base: core18
+version: git
+summary: 'Suspended Sentence: Point-and-click adventure game set on a space ship'
+description: |
+ You wake up to find yourself alone on a broken spaceship headed to a Penal
+ Colony. The ship seems to want you to fix it, but first you need to get out of
+ the room.
+ It's an adventure game, you know how this works.
+ Suspended Sentence was a PyWeek.org entry, a game written in a week.
+license: MIT
+
+grade: stable
+confinement: strict
+
+parts:
+  game:
+    plugin: python
+    python-version: python2
```

Comment:
Is there a specific reason we're not going with python 3 ?

Reply:
hrm, I didn't even try. It claimed 2.5 support, so... :P

Reply:
ah, pygame for python3 wasn't packaged in Ubuntu 18.04, it will only make it into the next LTS.

```
$ rmadison python3-pygame -u ubuntu
 python3-pygame | 1.9.4.post1+dfsg-3 | disco/universe | amd64, arm64, armhf, i386, ppc64el, s390x
 python3-pygame | 1.9.4.post1+dfsg-3 | eoan/universe  | amd64, arm64, armhf, i386, ppc64el, s390x
```

Probably not worth the pain of hacking around that until then.